### PR TITLE
Adding contexts 

### PR DESCRIPTION
**Pull request by VanDavv**
Now the flashcard and tip representation has URI fields which points to resource with image.

### CODE REVIEW

File: backend/app/src/main/java/com/bls/patronage/resources/TipResource.java [hunk 1]
```diff
@@ -49,4 +63,23 @@ public TipRepresentation updateTip(@Valid @PathParam("tipId") UUIDParam tipId,
 
         return tip;
     }
+
+    @Path("/essenceImage")
+    @POST
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    public TipRepresentation postEssenceImage(
```
Comment (mlotysz) [inline backend/app/src/main/java/com/bls/patronage/resources/TipResource.java]: Is this copy and paste from `FlashcardRepresentation`? Not good. Could you do it without copy? :)

**Comment by mlotysz** [comment]
OK. Not bad - names are good, finally! :)
1. BUT when we delete entities which has some storage objects I would like to see that those objects are deleted too.
2. BUT What about updating those objects?

File: backend/app/src/main/java/com/bls/patronage/resources/TipResource.java (hunk 1 above)
Comment (VanDavv) [inline backend/app/src/main/java/com/bls/patronage/resources/TipResource.java]: It's not copy and paste, although they look pretty the same. I was thinking about moving common code to separate function, but after all, main body consists of 4 lines, from which I could export 2 to separate method, and for me it was not necessary. However, I can do it

Comment (mlotysz) [inline backend/app/src/main/java/com/bls/patronage/resources/TipResource.java]: OK. Let's skip it. Fix major problems instead.

**Comment by mlotysz** [comment]
Thanks!
Missing func. will be delivered as separate PR.

**Pull request merged**